### PR TITLE
[new price plan] [dashboard] Update welcome section

### PR DESCRIPTION
**Pull request by csubira**
https://github.com/CartoDB/cartodb/issues/15434

https://projects.invisionapp.com/share/REVRCLQ38VT#/screens/402655763

### CODE REVIEW

File: lib/assets/javascripts/new-dashboard/core/constants/accounts.js [hunk 1]
```diff
@@ -1,8 +1,8 @@
-export const accountsWithLimits = ['Individual', 'Annual Individual'];
-export const freeAccountsWithLimits = ['Free 2020'];
+export const individual = ['Individual', 'Annual Individual'];
```
Comment (csubira) [inline lib/assets/javascripts/new-dashboard/core/constants/accounts.js]: Such a great solution to organize the different limit options!

File: lib/assets/javascripts/new-dashboard/core/constants/accounts.js [hunk 2]
```diff
@@ -1,8 +1,8 @@
...
-export const accountsWithTableLimits = [...accountsWithLimits, ...freeAccountsWithLimits];
-export const accountsWithPublicMapLimits = [...accountsWithLimits, ...freeAccountsWithLimits];
-export const accountsWithPrivateMapsLimits = [...accountsWithLimits, ...freeAccountsWithLimits];
-export const accountsWithApiKeysLimits = [...accountsWithLimits, ...freeAccountsWithLimits];
-export const accountsWithApiKeysLimitsToZero = freeAccountsWithLimits;
+export const accountsWithTableLimits = [...individual, ...free2020];
+export const accountsWithPublicMapLimits = [...individual, ...free2020];
+export const accountsWithPrivateMapsLimits = [...individual, ...free2020];
+export const accountsWithApiKeysLimits = [...individual, ...free2020];
+export const accountsWithApiKeysLimitsToZero = free2020;
```
Comment (csubira) [inline lib/assets/javascripts/new-dashboard/core/constants/accounts.js]: Shouldn't it be an array too?

Comment (csubira) [reply lib/assets/javascripts/new-dashboard/core/constants/accounts.js]: It is, so sorry! Disregard!

Comment (alejandraarri) [reply lib/assets/javascripts/new-dashboard/core/constants/accounts.js]: As there is only one item, so it's the same array, I chose to leave it like this. Maybe if we need to expand in the future we can change into an array.

**Pull request merged**